apis/mutations/blogs/useAddNewPost.ts

```diff
@@ -1,23 +1,40 @@
 'use client';
 
-import { useMutation } from '@tanstack/react-query';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
+import { toast } from 'react-toastify';
 import { useAxios } from '../../AxiosProvider';
 import ENDPOINTS from '../../endpoints';
+import QUERY_KEYS from '../../queryKeys';
 
 type NewPost = {
   title: string;
-  body: string;
-  userId: number;
+  excerpt: string;
+  content: string;
+  tags: number[] | string[];
```

Comment:
I would assume tags should be only string[]. What is the usecase for number[]?

Reply:
Apparently, no usecase.. for some reason I thought since user can send either list of integers or a list of strings, I should enable that. However, thinking about it now, there is no need for this. Will modify.